core/src/test/scala/org/dbpedia/extraction/mappings/DraftMappingExtractorTest.scala

```diff
@@ -0,0 +1,337 @@
+package org.dbpedia.extraction.mappings
+
+import java.io.File
+
+import org.dbpedia.extraction.destinations.Quad
+import org.dbpedia.extraction.ontology.io.OntologyReader
+import org.dbpedia.extraction.ontology.{OntologyProperty, Ontology}
+import org.scalatest.{PrivateMethodTester, FlatSpec}
+import org.scalatest.Matchers
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import scala.collection.mutable.{Set => MutableSet, HashSet}
+import org.dbpedia.extraction.sources.{XMLSource, WikiPage, MemorySource}
+import org.dbpedia.extraction.wikiparser.{WikiParser, Namespace, WikiTitle}
+import org.dbpedia.extraction.util.Language
+
+/**
+  *
+  */
+@RunWith(classOf[JUnitRunner])
+class DraftMappingExtractorTest extends FlatSpec with Matchers with PrivateMethodTester {
+
+  "DraftMappingExtractor" should """return correct property id's P17, P18, P94, P41""" in {
+
+    val lang = Language.English
+
+    val quads = parse(
+      """{{YM
+            | |status                                 =
+            | |azərbaycan dilində adı       = Telqte
+            | |orijinal adı                          = Telgte
+            | |ölkə                                    = {{#property:P17}}
+            | |şəkil                                    ={{#property:P18}}
+            | |gerb                                    = {{#property:P94}}
+            | |bayraq                                = {{#property:P41}}
+            | |bayraq yazısı                      =
+            |  |lat_dir =N |lat_deg =51 |lat_min =58 |lat_sec =55
+            |  |lon_dir =E |lon_deg =7 |lon_min =47 |lon_sec = 8
+            | |ölkə xəritəsi                        = <!-- alternativ, eyni koordinatlı diyarlar -->
+            | |statuslu                             =
+            | |sahəsi                               = 90.6
+            | |əhalisi                              = 19522
+            | |saat qurşağı                    = +1
+            | |telefon kodu                    = {{#property:P473}}
+            | |nəqliyyat kodu                 = {{#property:P395}}
+            | |sayt                                 = {{#property:P856}}
+            | |saytın dili                         = de
+            |}}
+            |""", "TestPage", lang)
+
+
+      val answer = Set("{{#property:P17}}","{{#property:P18}}","{{#property:P94}}", "{{#property:P41}}","{{#property:P473}}","{{#property:P395}}","{{#property:P856}}")
```

Comment:
This part is repeated in every case, should be converted in a function.
See for example https://github.com/aditya-nambiar/extraction-framework/blob/90a04aab1f944465086032fd2826b4f22bfb90b5/core/src/test/scala/org/dbpedia/extraction/dataparser/DateTimeParserTest.scala